LobbyHelper.cs

```diff
@@ -69,6 +70,28 @@ public static class LobbyHelper {
             return null;
         }
 
+        public static int GetLobbyIndex(string sid) {
```

Comment:
okay don't be mad!! but I'm wondering if tracking the lobby index is even needed anymore? it's only really used when determined default hearts (which I had a separate comment on altering). for the actual lobby selection you can just sort the SIDs alphabetically